vpp/vaapipostprocess_scaler.cpp

```diff
@@ -18,12 +18,15 @@
 #endif
 
 #include "vaapipostprocess_scaler.h"
```

Comment:
Put the denosie/de-interlance/... in vaapipostprocess_scaler.cpp/vaapipostprocess_scaler.h maybe is not good idea.